linkerd/proxy/client-policy/src/opaq.rs

```diff
@@ -127,6 +127,9 @@ pub(crate) mod proto {
             filters: NO_FILTERS.clone(),
             failure_policy: NonIoErrors::default(),
             distribution,
+            // XXX(eliza): maybe this shouldn't even be present on opaque
+            // routes...might have to turn it into a filter...
```

Comment:
What are we supposed to do with this comment? Should it be a filter? Or is this okay?

Reply:
What does a request timeout mean on an opaque route?

Reply:
https://github.com/linkerd/linkerd2-proxy/pull/2418/commits/26c9492af1e52ab85b9e1dcbeeed9b791e81dcbd changes this comment to simply state that the timeout is ignored on opaque routes, and adds a doc comment to `ClientPolicy` explaining the rationale behind that decision.